### ISSUE TITLE
V4 set target accepts #t=1.1 but does not accept #t=1.1,2.2

### ISSUE THREAD

**Issue by benwbrum** [UniversalViewer/universalviewer#827]
**UV version:**

```
 universalviewer@v4.0.0-pre.74
```

**I'm submitting a:**

- [x] bug report
- [ ] feature request => please use the [user stories repo](https://github.com/UniversalViewer/user-stories)
- [ ] support request => Please do not submit support requests here, use [stackoverflow](http://stackoverflow.com/questions/tagged/iiif)

**Current behavior:**

If `uv.set` is called with a time fragment containing a single point, UV will play the canvas at that point, as expected.  However, calling `uv.set` with a range fragment causes the following error:

```
Uncaught TypeError: Failed to set the 'currentTime' property on 'HTMLMediaElement': The provided double value is non-finite.
    at HTMLAudioElement.u.<computed> [as setCurrentTime]
```

**Expected behavior:**

Calling set with a fragment should play from the beginning of the range, stopping at the end of the range.

**Steps to reproduce:**

This is duplicated in this sandbox: https://codesandbox.io/s/uv-set-target-example-pn504?file=/index.html

**Related code:**
Real-world target example:
```
https://sizemoec.github.io/FFPC-FF010-FFPC_OVERVIEW_PT1/ff010/canvas-1/canvas#t=774.851273,794.572929
```

**Comment by edsilv** [comment]
Could we fix this for now so that if you include both values it doesn't break?

**Comment by benwbrum** [comment]
Absolutely.  We'd really love to support stop times, but that at least will keep things from breaking.

**Comment by edsilv** [comment]
That should be it patched for now to prevent breaking: https://github.com/UniversalViewer/universalviewer/commit/f7786ddfdff4ffc80544b8b424b948736b3950f9#diff-cb64973a8a5a2e0eac4714d6ee79b3016b2f2e7b76e35a803c9f28b4bf197b16R41-R45

**Comment by LlGC-szw** [comment]
All issues will be triaged for further investigation or closure by the 28 September 2023. If your issue is still relevant and would like for it be investigated further please comment by 14 September 2023.

**Comment by benwbrum** [comment]
This is still relevant.
